style/swift/sample.swift

```diff
@@ -85,3 +85,13 @@ struct DopeObject {
     return DopeObject(json: json)
   }
 }
+
+// Break up long function calls and initializers
+let foo = Foo(
+  bar: Bar(),
+  baz: Baz(),
+  quz: Quz(
+    x: 42,
+    y: 365
+  )
```

Comment:
I think if this is moved into a section specifically about style recommendations for breaking up long lines (as I intend to open a PR for shortly) it will be clear that this isn't a hard rule for all functions.